### PR TITLE
fix: Updated the Icons Markup & CSS as per the UI/UX perspective, also fixed the Reset not working properly for ast-color, ast-slider controls.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - Trello task - https://trello.com/c/idwR1QsN/1107-change-reset-icon-design

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Styling Changes in CSS.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 1]
```diff
@@ -2357,7 +2372,21 @@ input[type="text"].select2-search__field {
...
+  left: unset;
+}
+.customize-control-ast-responsive-color .ast-responsive-btns {
+  position: absolute;
+  left: 42px;
+  top: 5.7px;
+}
+.ast-field-settings-modal .customize-control-ast-responsive-color .ast-responsive-btns  {
+  left: 8px;
+}
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:2389]: @imnavanath Please check if we can combine the above CSS with this?

`.customize-control-ast-responsive-background .ast-color-btn-reset-wrap {`

Both have same CSS properties.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css [hunk 2]
```diff
@@ -3785,13 +3828,28 @@ input[type="text"].select2-search__field {
...
+  left: unset;
+}
+.customize-control-ast-responsive-background .ast-responsive-btns {
+  position: absolute;
+  left: 42px;
+  top: 5.7px;
+}
+.ast-field-settings-modal .customize-control-ast-responsive-background .ast-responsive-btns  {
+  left: 8px;
+}
```
Comment (premanshup) [inline inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:3845]: @imnavanath This looks like a duplicate CSS - 

https://github.com/brainstormforce/astra/pull/2337/files#diff-168566eaa47d45cb74bf319565059c0a924f9c87e66451821f796d041fda9377R2280

Please make sure there are no other duplicate CSS.

File: inc/customizer/extend-custom-controls/src/color/color-component.js [hunk 3]
```diff
@@ -29,12 +29,23 @@ const ColorComponent = props => {
...
+		
+		let resetFlag = true;
+		let tempVal = state.value.replace( 'unset', '' );
+
+		if ( JSON.stringify(tempVal) !== JSON.stringify(defaultValue) ) {
+			resetFlag = false;
+		}
+		console.log(props.control);
+		console.log(tempVal);
+		console.log(defaultValue);
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/color/color-component.js:41]: @imnavanath Remove all `console.log()`

File: inc/customizer/extend-custom-controls/src/color/color-component.js [hunk 4]
```diff
@@ -29,12 +29,23 @@ const ColorComponent = props => {
 	};
 
 	const renderOperationButtons = () => {
+		
+		let resetFlag = true;
+		let tempVal = state.value.replace( 'unset', '' );
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/color/color-component.js:34]: @imnavanath This can be `const` instead of `let`

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 1 above)
Comment (RajkiranBagal) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css:2389]: Done.

File: inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css (hunk 2 above)
Comment (RajkiranBagal) [reply inc/customizer/custom-controls/assets/css/unminified/custom-controls-rtl.css]: Removed Duplicate CSS, & checked for no other.

File: inc/customizer/extend-custom-controls/src/color/color-component.js (hunk 3 above)
Comment (RajkiranBagal) [reply inc/customizer/extend-custom-controls/src/color/color-component.js]: Removed.

File: inc/customizer/extend-custom-controls/src/color/color-component.js (hunk 4 above)
Comment (RajkiranBagal) [reply inc/customizer/extend-custom-controls/src/color/color-component.js]: Done.

**Review by premanshup — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**